app/Models/Beatmapset.php

```diff
@@ -19,6 +19,7 @@
  */
 namespace App\Models;
 
+use App;
```

Comment:
This isn't used.

(I thought styleci caught this already but apparently it didn't)